allofplos/tests/test_partial_dois.py

```diff
@@ -0,0 +1,52 @@
+from .. import Corpus, Article, starterdir
+from ..plos_regex import validate_partial_doi, validate_doi
+from ..transformations import partial_to_doi, doi_to_partial
+
+import pytest
+
+
+@pytest.fixture
+def corpus():
+    return Corpus(starterdir, seed=1000)
+
+
+@pytest.fixture
+def test_article():
+    return Article('10.1371/journal.pone.0040259', directory=starterdir)
+
+
+@pytest.fixture
+def test_doi():
+    return '10.1371/journal.pone.0040259'
+
+
+@pytest.fixture
+def test_partial_doi():
+    return 'pone.0040259'
+
+
+def test_partial_doi_regex(test_partial_doi):
```

Comment:
Could you test the regexes directly as well?

Reply:
Actually this test is confusingly named… its not testing the doi regexes it's testing the validate_partial_doi function.